lib/bugsnag/integrations/railtie.rb

```diff
@@ -73,9 +74,15 @@ class Railtie < ::Rails::Railtie
     # @api private
     # @param event [Hash] details of the event to subscribe to
     def event_subscription(event)
-      ActiveSupport::Notifications.subscribe(event[:id]) do |*, data|
+      ActiveSupport::Notifications.subscribe(event[:id]) do |*, event_id, data|
         filtered_data = data.slice(*event[:allowed_data])
         filtered_data[:event_name] = event[:id]
+        filtered_data[:event_id] = event_id
+        if event[:id] == "sql.active_record"
+          binds = data[:binds].map { |bind| defined?(bind.name) ? [bind.name, '?'] : nil }
+          binds.select! { |bind| bind }
+          filtered_data[:bindings] = JSON.dump(binds.to_h) unless binds.empty?
```

Comment:
I think we should call this `:binds` instead of `:bindings` to be consistent with ActiveRecord's naming.